zds/utils/tutorials.py

```diff
@@ -64,10 +68,11 @@ def export_part(part):
 
 
 def export_tutorial(tutorial):
-    from zds.tutorial.models import Part, Chapter
-    '''
+    """
     Export a tutorial to a dict
-    '''
+    """
+
+    from zds.tutorial.models import Part, Chapter
```

Comment:
on which line do you want to me to add a blank line?
I'm guessing you meant line 76 but that one is already a blank line.